shared/api/src/lib.rs

```diff
@@ -180,6 +182,88 @@ impl Class {
             _ => Class::Unknown,
         }
     }
+
+    /// Returns base stats for a class of ship
+    pub fn default_stats(&self) -> ClassStats {
+        match self {
+            Class::Fighter => ClassStats {
+                max_health: 100.0,
+                mass: 15000.0,
+                max_forward_acceleration: 60.0,
+                max_backward_acceleration: 30.0,
+                max_lateral_acceleration: 30.0,
+                max_angular_acceleration: TAU,
+            },
+            Class::Frigate => ClassStats {
+                max_health: 10000.0,
+                mass: 4e6,
+                max_forward_acceleration: 10.0,
+                max_backward_acceleration: 5.0,
+                max_lateral_acceleration: 5.0,
+                max_angular_acceleration: TAU / 8.0,
+            },
+            Class::Cruiser => ClassStats {
+                max_health: 20000.0,
+                mass: 9e6,
+                max_forward_acceleration: 5.0,
+                max_backward_acceleration: 2.5,
+                max_lateral_acceleration: 2.5,
+                max_angular_acceleration: TAU / 16.0,
+            },
+            Class::Asteroid => ClassStats {
+                max_health: 200.0,
+                mass: 20e6,
+                max_forward_acceleration: 0.0,
+                max_backward_acceleration: 0.0,
+                max_lateral_acceleration: 0.0,
+                max_angular_acceleration: 0.0,
+            },
+            Class::Target => ClassStats {
+                max_health: 1.0,
+                mass: 10.0,
+                max_forward_acceleration: 0.0,
+                max_backward_acceleration: 0.0,
+                max_lateral_acceleration: 0.0,
+                max_angular_acceleration: 0.0,
+            },
+            Class::Missile => ClassStats {
+                max_health: 20.0,
+                mass: 150.0,
+                max_forward_acceleration: 300.0,
+                max_backward_acceleration: 0.0,
+                max_lateral_acceleration: 100.0,
+                max_angular_acceleration: 4.0 * TAU,
+            },
+            Class::Torpedo => ClassStats {
+                max_health: 100.0,
+                mass: 500.0,
+                max_forward_acceleration: 70.0,
+                max_backward_acceleration: 0.0,
+                max_lateral_acceleration: 20.0,
+                max_angular_acceleration: 2.0 * TAU,
+            },
+            Class::Unknown => ClassStats {
+                max_health: 100.0,
+                mass: 1000.0,
+                max_forward_acceleration: 0.0,
+                max_backward_acceleration: 0.0,
+                max_lateral_acceleration: 0.0,
+                max_angular_acceleration: 0.0,
+            },
+        }
+    }
+}
+
+/// Stats for a class of ship
+#[derive(Debug, Clone)]
+#[allow(missing_docs)]
+pub struct ClassStats {
```

Comment:
Let's plan on adding stats for weapons, radar, radio, etc to this class in the future. Given that, it wouldn't really make sense for this class to be used as "base_stats" in Ship anymore. So I think we should copy out the values from this class to Ship instead.

Fortunately I think we can just make a function `ClassStats -> Ship` and use that in ship::fighter/etc so there shouldn't be too much boilerplate.

Reply:
Oh yeah I see what you're saying. I'll make that change.